language-support/hs/bindings/examples/nim/package.yaml

```diff
@@ -0,0 +1,37 @@
+name:                nim
+version:             0.1.0.0
+description:         Play Nim on a DAML Ledger
+license:             BSD3
```

Comment:
This section is not right. Our code is licensed as `Apache2`, author should probably be `The DAML Authors` like in our copyright header and the maintainer email should probably point to a digitalasset email address. The repo can just point to the DAML repo.

Same for the other cabal files.

Reply:
changed as you suggested